pkg/apis/v1beta1/nodepool.go

```diff
@@ -155,10 +155,19 @@ type NodePoolList struct {
 	Items           []NodePool `json:"items"`
 }
 
-// OrderByWeight orders the nodepools in the NodePoolList
-// by their priority weight in-place
+// OrderByWeight orders the NodePools in the NodePoolList by their priority weight in-place.
+// This priority evaluates the following things in precedence order:
+//  1. NodePools that have a larger weight are ordered first
+//  3. If two NodePools have the same weight, then the NodePool with the comparatively larger name will come first
```

Comment:
Can you fix this comment with the numbering?

Additionally, as a clarifying point, does comparatively larger mean earlier or later in the alphabet?